inc/saiswitch.h

```diff
@@ -2265,6 +2265,14 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_TUNNEL_LOOPBACK_PACKET_ACTION,
 
+    /**
+     * @brief Available MPLS in segment entries
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_AVAILABLE_MPLS_INSEG_ENTRY,
+
```

Comment:
Please consider using sai_object_type_get_availability()
This is generic object CRM API, replacing the old approach of specific attribute per object

Reply:
Is there an example of this that I can see?  I don't see any usage of sai_object_type_get_availability() in latest CrmOrch.cpp.

Reply:
The PR has examples
https://github.com/opencomputeproject/SAI/commit/ef89cb20a09aa846d06ff2e5d2d3035afd3dee07#diff-1f169362ad7823ade5ade3918ba6a35fe17189e13d7f05a8b9edf2f6aeccc5f8

Reply:
AFAIK, Sonic hasn't yet ported existing CRM code to the new API, but is using the new API for new objects, such as debug counters

Reply:
https://github.com/opencomputeproject/SAI/commit/ef89cb20a09aa846d06ff2e5d2d3035afd3dee07#diff-1f169362ad7823ade5ade3918ba6a35fe17189e13d7f05a8b9edf2f6aeccc5f8
Shows usage examples